04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,25 @@
-#write your code here
+# Rule 1: If a word begins with a vowel sound, add an 
+#"ay" sound to the end of the word.
+#
+# Rule 2: If a word begins with a consonant sound, move 
+#it to the end of the word, and then add an "ay" sound 
+#to the end of the word.
+
+
+def translate(str)
+    str.gsub!(/\w+/) { |word| translate_word(word)}
+end
+
+def translate_word(phrase)
+    phrase.split(" ").map do |word|
+        word = word.gsub("qu", " ")
+        word.gsub!(/^([^aeiou]*)(.*)/,'\2\1ay')
```

Comment:
this regex is a bit complex.  I might consider breaking it into two cases, rather than solving it in just one.  Just my personal preference though.  Or maybe a comment describing how it works